instrumentation/http-url-connection/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/httpurlconnection/HttpUrlConnectionTest.java

```diff
@@ -280,20 +293,68 @@ public void getOutputStreamShouldTransformGetIntoPost() throws IOException {
                         .hasKind(CLIENT)
                         .hasParent(trace.getSpan(0))
                         .hasAttributesSatisfyingExactly(
-                            equalTo(SemanticAttributes.NET_PROTOCOL_NAME, "http"),
-                            equalTo(SemanticAttributes.NET_PROTOCOL_VERSION, "1.1"),
-                            equalTo(SemanticAttributes.NET_PEER_NAME, "localhost"),
-                            equalTo(SemanticAttributes.NET_PEER_PORT, url.getPort()),
-                            equalTo(SemanticAttributes.HTTP_URL, url.toString()),
-                            equalTo(SemanticAttributes.HTTP_METHOD, "POST"),
-                            equalTo(SemanticAttributes.HTTP_STATUS_CODE, STATUS),
+                            equalTo(getAttributeKey(SemanticAttributes.NET_PROTOCOL_NAME), "http"),
+                            equalTo(
+                                getAttributeKey(SemanticAttributes.NET_PROTOCOL_VERSION), "1.1"),
+                            equalTo(getAttributeKey(SemanticAttributes.NET_PEER_NAME), "localhost"),
+                            equalTo(
+                                getAttributeKey(SemanticAttributes.NET_PEER_PORT), url.getPort()),
+                            equalTo(getAttributeKey(SemanticAttributes.HTTP_URL), url.toString()),
+                            equalTo(getAttributeKey(SemanticAttributes.HTTP_METHOD), "POST"),
+                            equalTo(getAttributeKey(SemanticAttributes.HTTP_STATUS_CODE), STATUS),
                             satisfies(
-                                SemanticAttributes.HTTP_REQUEST_CONTENT_LENGTH,
+                                getAttributeKey(SemanticAttributes.HTTP_REQUEST_CONTENT_LENGTH),
                                 AbstractLongAssert::isNotNegative),
                             satisfies(
-                                SemanticAttributes.HTTP_RESPONSE_CONTENT_LENGTH,
+                                getAttributeKey(SemanticAttributes.HTTP_RESPONSE_CONTENT_LENGTH),
                                 AbstractLongAssert::isNotNegative)),
                 span ->
                     span.hasName("test-http-server").hasKind(SERVER).hasParent(trace.getSpan(1))));
   }
+
+  @ParameterizedTest
+  @ValueSource(strings = {"http", "https"})
+  public void traceRequestWithConnectionFailure(String scheme) {
```

Comment:
This test was previously in a separate class, deleted it and moved the test here.